From 3cb1cd9c99018dd09ed57b49044e136e6e11dde9 Mon Sep 17 00:00:00 2001
From: Jani Hautakangas <jani@kodegood.com>
Date: Sat, 16 Aug 2025 13:18:36 +0300
Subject: [PATCH] Debianize clang toolchain paths

---
 build/config/clang/BUILD.gn | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/build/config/clang/BUILD.gn b/build/config/clang/BUILD.gn
index 4b6365dc70..10dc3f3939 100644
--- a/build/config/clang/BUILD.gn
+++ b/build/config/clang/BUILD.gn
@@ -209,13 +209,15 @@ template("clang_lib") {
         }
       } else if (is_linux || is_chromeos) {
         if (current_cpu == "x64") {
-          _dir = "x86_64-unknown-linux-gnu"
+          _dir = "linux"
+          _suffix = "-x86_64"
         } else if (current_cpu == "x86") {
           _dir = "i386-unknown-linux-gnu"
         } else if (current_cpu == "arm") {
           _dir = "armv7-unknown-linux-gnueabihf"
         } else if (current_cpu == "arm64") {
-          _dir = "aarch64-unknown-linux-gnu"
+          _dir = "linux"
+          _suffix = "-aarch64"
         } else if (current_cpu == "loong64") {
           _dir = "loongarch64-unknown-linux-gnu"
         } else if (current_cpu == "riscv64") {
-- 
2.48.1

